packages/react/src/Button/Button.tsx

```diff
@@ -0,0 +1,47 @@
+/**
+ * @license EUPL-1.2+
+ * Copyright (c) 2021 Robbert Broersma
+ * Copyright (c) 2023 Gemeente Amsterdam
+ */
+
+import { Button as CommunityButton } from '@utrecht/component-library-react'
+import { ButtonHTMLAttributes, ForwardedRef, forwardRef, PropsWithChildren } from 'react'
+
+export interface ButtonProps extends ButtonHTMLAttributes<HTMLButtonElement> {
+  // TODO: candidate for being its own type
+  variant?: 'primary' | 'secondary' | 'tertiary'
+}
+
+type CommunityButtonAppearance = 'primary-action-button' | 'secondary-action-button' | 'subtle-button'
+
+function getAppearance(variant: ButtonProps['variant']): CommunityButtonAppearance {
+  switch (variant) {
+    case 'secondary':
+      return 'secondary-action-button'
+    case 'tertiary':
+      return 'subtle-button'
+    default:
+      return 'primary-action-button'
+  }
+}
+
+export const Button = forwardRef(
+  (
+    { children, disabled, variant = 'primary', ...rest }: PropsWithChildren<ButtonProps>,
+    ref: ForwardedRef<HTMLButtonElement>,
+  ) => {
+    return (
+      <CommunityButton
+        {...rest}
+        appearance={getAppearance(variant)}
+        ref={ref}
+        aria-disabled={disabled ?? false}
+        disabled={disabled}
```

Comment:
This is not necessary for MVP, we can finetune this in [a seperate ticket](https://gemeente-amsterdam.atlassian.net/browse/DES-156)

Reply:
done